ganalytics.php

```diff
@@ -67,7 +67,7 @@ private function getTrackingCode($scriptName, $objectName, $async=false)
                 "(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){\n".
                 "(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),\n".
                 "m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)\n".
-                "})(window,document,'script','//www.google-analytics.com/{$scriptName}.js','{$objectName}');\n"
+                "})(window,document,'script','https://www.google-analytics.com/{$scriptName}.js','{$objectName}');\n"
```

Comment:
Because GA can handle HTTP and HTTPS, it would be nice, if you create a configurable parameter "protocol" with the options "http", "https" and "auto". So nobody is forced to use HTTP or HTTPS.